tests/tests/cpld_bmark_p7

```diff
@@ -50,17 +51,17 @@ export LIST_FILES="sfcf006.nc \
 export_fv3
 export_cpl
 
-export SYEAR="2013"
-export SMONTH="04"
-export SDAY="01"
-export SHOUR="00"
+export SYEAR=2013
+export SMONTH=04
+export SDAY=01
+export SHOUR=00
+export SECS=`expr $SHOUR \* 3600`
+export BMIC=.true.
 
-export DAYS='0.25'
-export FHMAX='6'
-export RESTART_N="3"
```

Comment:
Isn't RESTART_N = FHMAX - FHROT? I think we need to be consistent throughout standalone, coupled, and etc.

Reply:
This test is actually the previous bmark (wave) test. So we're not restarting it, just writing a restart at the end of the run. Since FHMAX has changed, RESTART_N needs to be updated.

Reply:
Thanks for the explanation. In this case FHROT=0. So, I assume FHMAX-FHROT should not make a difference. I just thought being consistent may be beneficial because it can cause confusion down the line.

Reply:
I see, I can make it FHMAX-FHROT.